### DISCUSSION TITLE
Custom Query with $and (aggregation)

### DISCUSSION THREAD

**Discussion by maximiliano-gt** [strapi/strapi#8441] [Legacy Topics]
Hello !! I'm trying to make this simple custom query:

`find(ctx) {`
`                      return strapi.query("geoinfo").find({`
`      $and: [{ latitude: { $gt: ctx.lat } }, { longitude: { $gt: ctx.lng } }],`
`    });`
`  }`

But im getting this 400 error: 

>  "message": "Your filters contain a field '$and' that doesn't appear on your model definition nor it's relations"

Which is the right way to do this? Thank you !

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

**Reply by maximiliano-gt** [reply]
thank you, I posted it in the forum!
